manifests/t/Trufflesuite/Ganache/Beta/2.6.0-beta.3/Trufflesuite.Ganache.Beta.locale.en-US.yaml

```
@@ -0,0 +1,18 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Trufflesuite.Ganache.Beta
+PackageVersion: 2.6.0-beta.3
+PackageLocale: en-US
+Publisher: Truffle
+PublisherUrl: https://www.trufflesuite.com
+PublisherSupportUrl: https://www.trufflesuite.com/docs/ganache/overview
+PrivacyUrl: https://www.trufflesuite.com/policy
+PackageName: Ganache
```

```suggestion
PackageName: Ganache (Beta)
```
to avoid conflicts with the stable version which would cause issues with `winget export`.